src/bijectors/exp_log.jl

```diff
@@ -8,8 +8,11 @@ struct Log{N} <: Bijector{N} end
 Exp() = Exp{0}()
 Log() = Log{0}()
 
-(b::Exp)(y) = @. exp(y)
-(b::Log)(x) = @. log(x)
+(b::Exp)(y::Real) = exp(y)
```

Comment:
I actually added this specification because I saw a performance-degradation when using `exp.(x)` when `x isa Real` compared to `exp(x)`:
```julia
julia> using BenchmarkTools

julia> x = 1.0
1.0

julia> @btime exp($x)
  0.020 ns (0 allocations: 0 bytes)
2.718281828459045

julia> @btime exp.($x)
  2.845 ns (0 allocations: 0 bytes)
2.718281828459045
```

Worth noting: this is on Julia 1.0! On Julia 1.3:
```julia
julia> using BenchmarkTools

julia> x = 1.0
1.0

julia> @btime exp($x)
  0.020 ns (0 allocations: 0 bytes)
2.718281828459045

julia> @btime exp.($x)
  0.020 ns (0 allocations: 0 bytes)
2.718281828459045
```

Reply:
Fair enough.